lib/api/v3/work_packages/work_package_representer.rb

```diff
@@ -231,9 +239,16 @@ class WorkPackageRepresenter < ::API::Decorators::Single
                  },
                  setter: -> (value, *) { represented.description = value['raw'] },
                  render_nil: true
-        property :priority, getter: -> (*) { priority.try(:name) }, render_nil: true
```

Comment:
@0xF013, @NobodysNightmare  this is going to destroy displaying the priority in the WP pane.